Cargo.toml

```diff
@@ -42,7 +42,7 @@ dirs = "4"
 dotenv = "0.15.0"
 ethers = "1.0.2"
 integer-encoding = "3.0.4"
-ipld_traversal = { git = "https://github.com/kckeiks/rs-graphsync.git", rev = "b8728c7ad02aa42cc11954192e95875f14fc6380" }
```

Comment:
`rs-graphsync` does not send the request_id on an error so I had to update our fork.